### ISSUE TITLE
tuareg 2.0.8 doesn't install anything

### ISSUE THREAD

**Issue by ivg** [ocaml/opam-repository#3643]
new `opam` file for the tuareg 2.0.8 doesn't perfrom installation

**Comment by Chris00** [comment]
It does.  It is using `opam/files/tuareg.install` for that.

**Comment by Chris00** [comment]
(I mean the newer versions.)

**Comment by ivg** [comment]
Sure, it installs on opam 1.2, but this doesn't work on opam 1.1

**Comment by ivg** [comment]
This is how `opam` file looks in old opam:

```
opam-version: "1.1"
maintainer: "Christophe.Troestler@umons.ac.be"
author: [
  "Albert Cohen <Albert.Cohen@prism.uvsq.fr>"
  "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
]
homepage: "https://forge.ocamlcore.org/projects/tuareg/"
bug-reports: "https://forge.ocamlcore.org/tracker/?group_id=43"
build: [make "tuareg-site-file.el"]
post-messages: "\nIf you have not yet done so, please add in ~/.emacs.d/init.el or\nin ~/.emacs to following line:\n    (load \"%{share}%/emacs/site-lisp/tuareg-site-file\")\n  " {success}
```

**Comment by ivg** [comment]
and 
```

**Comment by cakeplus** [comment]
Can confirm the problem: installing `tuareg` doesn't work in OPAM 1.1.1 on Ubuntu 14.04 (stock `opam` package).

**Comment by Gugurumbe** [comment]
Opam 1.2 doesn't install tuareg.2.0.9 properly either (missing tuareg-site-file), on Debian stable (with 1.2 source tarball at http://opam.ocaml.org/doc/Install.html)

**Comment by Chris00** [comment]
Thanks for reporting.  I think this is fixed by https://github.com/ocaml/opam-repository/pull/3798 (the file `tuareg.install` was missing).

**Comment by yallop** [comment]
I'm closing this, since it's apparently fixed by #3798.  Please reopen the issue if the problem reoccurs.
